package.json

```diff
@@ -116,18 +115,13 @@
     ]
   },
   "lint-staged": {
-    "*.js": [
+    "*.{js,ts}": [
```

Comment:
ESLint didn't support TypeScript when this was first added in 2018, but it does now!